CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
 
+## 9.5.0-alpha.1 - September 3, 2020
```

Comment:
```suggestion
## 9.5.0-alpha.1 - September 4, 2020
```

Reply:
Correct, but I would avoid to rebase - it'll breaks `android-internal`. :-)